examples/fault-injection/README.md

```diff
@@ -0,0 +1,44 @@
+Envoy fault injection example
+=============================
+
+In this example, we show [Envoy's fault injection
+feature](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/fault_filter) in a sandbox.
+
+## Usage
+```
+# in terminal 1
+docker-compose up
+
+# in terminal 2
+docker-compose exec envoy bash
+bash send_request.sh
+```
+
+This constantly sends HTTP GET requests to running Envoy process. The process proxies the requests to the backend server
```

Comment:
The script above (send_request.sh) sends a continuous stream of HTTP requests to Envoy, which in turn forwards the requests to the backend container. Fault injection is configured in Envoy but turned off (i.e. affects 0% of requests). Consequently, you should see a continuous sequence of HTTP 200 response code.